services/dolly/go-dolly

```diff
@@ -0,0 +1,17 @@
+#!/bin/ash
+
+timestamp=`date +%Y/%m/%d-%H:%M:%S`
+echo "go-dolly: I was called at $timestamp"
+
+config_path="/config/rclone/rclone.conf"
+data_path="/data/test"
+remote_name="remotebox"
+remote_path="isoblue_rclone"
+
+if [ ! -f $config_path ] || [ "$override" == "1" ]
+then
+  mkdir -p /config/rclone
+  echo -en "[$remote_name]\ntype = ${type}\nbox_sub_type = ${box_sub_type}\ntoken = ${token}" > $config_path
+fi
+
+rclone copy --progress --local-no-check-updated --compress-level=$clvl --checkers=$numcheckers --bwlimit=$bwlimit --log-level DEBUG $data_path $remote_name:$remote_path
```

Comment:
What if `$clvl`, `$bwlimit`, `$numcheckers` 

Do we always want --log-level DEBUG in production?

Reply:
`$clvl` is compression level. Set to 0 to disable compression.
`$bwlimit` is bandwidth limit in bytes/second.
`$numcheckers` is the number of checkers running in parallel for verifying the files on the server and the backend side. More the better but I set it to 2 because I suspect it was pegging the CPU.

Reply:
Sorry I wanted to say, "what if `$clvl`, `$bwlimit`, `$numcheckers` were not set". I think `rclone` likely dies with a help print out?

Seems like these are okay to have reasonable defaults for.